plugins/ipam/vnialloc/vnialloc.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package vnialloc;
+
+// CustomVniVxlanSFC represents vni allocation made for a custom vxlan made for an SFC
+message CustomVniVxlanSFC {
+    string sfc_name = 1;            // name of the service Function Chain
```

Comment:
As noted above, please make this more generic, not tied to SFCs, e.g.:
```proto
message VxlanVniAllocation {
    string vxlan_name= 1;   
    uint32 vni = 2;  
}
```

Reply:
done.